aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -87,18 +85,45 @@ public void PlaceTile(int x, int y, Tile tile)
         {
             GameObject prefab = tile switch
             {
-                Corner _ => RoomObjects.Corner,
-                Crossing _ => RoomObjects.Crossing,
-                DeadEnd _ => RoomObjects.DeadEnd,
-                Empty _ => RoomObjects.Empty,
-                Room _ => RoomObjects.Room,
-                Straight _ => RoomObjects.Straight,
-                TSection _ => RoomObjects.TSection,
-                _ => null
+                Corner   => RoomObjects.Corner,
+                Crossing => RoomObjects.Crossing,
+                DeadEnd  => RoomObjects.DeadEnd,
+                Empty    => RoomObjects.Empty,
+                Room     => RoomObjects.Room,
+                Straight => RoomObjects.Straight,
+                TSection => RoomObjects.TSection,
+                _        => null
```

Comment:
To keep the styling the same with the other statements:
```suggestion
                Corner => RoomObjects.Corner,
                Crossing => RoomObjects.Crossing,
                DeadEnd => RoomObjects.DeadEnd,
                Empty => RoomObjects.Empty,
                Room => RoomObjects.Room,
                Straight => RoomObjects.Straight,
                TSection => RoomObjects.TSection,
                _ => null
```